Testing/VelaptorTests/OpenGL/GLWindowTests.cs

```diff
@@ -669,10 +671,11 @@ public void MouseCursorVisible_WhenSettingValueAndNotCaching_ReturnsCorrectResul
     public void WindowState_WhenGettingInvalidValue_ThrowsException()
     {
         // Arrange
-        var expected = $"The value of the enum '{nameof(WindowState)}' used in the class '{nameof(GLWindow)}' and";
-        expected += " method 'SetupOtherPropCaches' is invalid and out of range.";
+        var invalidValue = 1234;
```

Comment:
Please refactor this to a constant

```suggestion
        const invalidValue = 1234;
```